babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -327,6 +327,34 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournalTable() {
```

Comment:
Maybe the name of this test could be?

testTableAttributeJournalTableWithSimpleIdentifier()